node/src/utils/work_queue.rs

```diff
@@ -138,6 +139,18 @@ impl<T> WorkQueue<T> {
         self.notify.notify_waiters();
     }
 
+    /// Creates a streaming consumer of the work queue.
+    #[inline]
```

Comment:
Why `#[inline]` here?

Reply:
So, in general, `#[inline]` does nothing when used inside the same crate. Since we are not using this in any way or form across crate boundaries, I could have omitted it here.

However, I do like to code things in `utils` like they could be ripped out and moved elsewhere at a moment's notice, so this is a "defensive" inline declaration, should this be moved.

`to_stream` is a function that calls just another function, so in general, I like those to be inlined in general, as they would disappear completely.

In this case I believe it does not matter at all, since the `impl Streal` makes it generic over an anonymous type parameter, AFAIK this means it will have to be monomorphized anyway and thus not be a non-inlineable cross-crate function.

So I guess the tl;dr is "for defensive reasons". We can remove it, as I assume it has 0 effect here, but it does no harm and if the function signature and location of this would ever change, we would want to re-add it.